rules/game_process.tex

```diff
@@ -75,7 +81,16 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
-The referee will announce the transition from the \texttt{initial} to \texttt{ready} state by raising both hands over its head. The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following year.}.
+At the scheduled time, the referee will announce that the transition from the \texttt{setup} to \texttt{initial} will be executed 60 seconds later.
+The referee shall call ``Initial in 60 seconds''. 
+The GameController will execute this transition automatically once the time has passed. 
```

Comment:
I don't think we should introduce a GC countdown here. There might be discussion between teams and the headref or GC operator. 

The transition from setup to inital simply happens when the GC decides.

Reply:
This was your suggestion in another comment unless I misunderstand. I liked that because it would help to have matchs starting in time and make more clear the time window teams are having to call a timeout. 

What we can do is keeping the countdown but the transition to be done manually by the GC operator.

Reply:
Does the suggestion of the countdown mean that the GameController should know when the match starts?

Reply:
I was thinking to rely on the referee to start on time. It would add so many issues if let the GC starting match without manual input.

Reply:
OK, I was reading this as if the countdown (even without automatic transition, which I consider a bad idea, too) should be displayed by the GC. If it's just the referee looking at a clock and announcing the time, it's fine.

Reply:
Just to be clear I mean the operator to start the initial countdown manually ( 60s before the transition to initial). The current discussion tends to also trigger the transition manually which I will update. 
There already a 1 minutes rules and 2 minutes rules about initial kick-off. This is just a matter of better defining when this start (Section 3.5). For a matter of clarity the GC should display this countdown.